types/got/index.d.ts

```diff
@@ -98,6 +87,36 @@ declare namespace got {
         (url: GotUrl, options: GotBodyOptions<null>): GotPromise<Buffer>;
     }
 
+    interface GotJSONFn {
```

Comment:
Why this interface is called `JSON`, but has overloads for `GotFormOptions` and `GotBodyOptions`?